### PR TITLE
Feature/dashboard layout

### PR DESCRIPTION
**Pull request by MaaxLav**
Dashboard layout

### CODE REVIEW

File: dtApi/src/app/modules/dashboard/components/statistics/statistics.component.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    layoutChangesMedia = this.breakpointObserver.observe([
+        this.ResponsiveData.isXLScreen,
+        this.ResponsiveData.isLGScreen,
+        this.ResponsiveData.isMDScreen,
+        this.ResponsiveData.isSMScreen,
+    ])
+    layoutChange(result: object): void {
+        for (const item in result) {
+            if (
+                item == this.ResponsiveData.isXLScreen &&
```
Comment (vroh17) [inline dtApi/src/app/modules/dashboard/components/statistics/statistics.component.ts:40]: always use ```===```

File: dtApi/src/app/modules/dashboard/services/dashboard-metrics.service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { HttpClient } from '@angular/common/http'
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs/internal/Observable'
+import { environment } from 'src/environments/environment'
+
+@Injectable()
+export class DashboardMetricsService {
+    private baseurl = environment.BASEURL
+    constructor(private httpInstance: HttpClient) {}
+    BackendLogin(): Observable<any> {
```
Comment (vroh17) [inline dtApi/src/app/modules/dashboard/services/dashboard-metrics.service.ts:12]: ```BackendLogin``` => ```backendLogin```

Comment (vroh17) [inline dtApi/src/app/modules/dashboard/services/dashboard-metrics.service.ts]: regular method names should start from lower case

File: dtApi/src/app/modules/dashboard/components/statistics/statistics.component.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        //     .BackendLogin()
+        //     .subscribe((info) => (this.logininfo = info))
+
+        {
+            this.infoservice
+                .getFacultiesNumber()
+                .subscribe(
+                    (facultiesInfo) =>
+                        (this.facultiesAmount = facultiesInfo.numberOfRecords)
+                )
```
Comment (vroh17) [inline dtApi/src/app/modules/dashboard/components/statistics/statistics.component.ts]: I would recommend to use async pipes in such cases. It reduces the amount of code significantly

Comment (vroh17) [reply dtApi/src/app/modules/dashboard/components/statistics/statistics.component.ts]: Lets discuss it on meeting

File: dtApi/src/app/modules/dashboard/services/dashboard-metrics.service.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        return this.httpInstance.post(`${this.baseurl}/api/Login/index`, {
+            username: 'admin',
+            password: 'dtapi_admin',
+        })
+    }
+    getFacultiesNumber(): Observable<any> {
+        return this.httpInstance.get(`${this.baseurl}/api/Faculty/countRecords`)
```
Comment (vroh17) [inline dtApi/src/app/modules/dashboard/services/dashboard-metrics.service.ts]: ```.pipe(map(res) => res.numberOfRecords)```
```.pipe(pluck('numberOfRecords')```

**Review by vroh17 — APPROVED** [review]

**Review by KvakSebastian — APPROVED** [review]
Nice

**Pull request merged**